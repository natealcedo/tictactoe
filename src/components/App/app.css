.app {
  background-color: white;
  min-height: calc(100vh - 5px);
  width: 100%;
  margin-top: 5px;
}

.header {
  color: #00363d;
}