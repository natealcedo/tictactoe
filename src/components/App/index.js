import React from "react";
import PlayerForm from "../PlayerForm";
import GameBoard from "../GameBoard";
import "./app.css";

const App = () => (
  <div className="app">
    <h1 className="header">Tic Tac Toe</h1>
    <PlayerForm />
    <GameBoard />
  </div>
);

export default App;
