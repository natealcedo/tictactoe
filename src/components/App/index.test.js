import React from "react";
import App from ".";
import { shallow } from "enzyme";

describe("App", () => {
  test("Should render without exploding", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
