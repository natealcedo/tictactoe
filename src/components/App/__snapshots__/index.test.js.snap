// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render without exploding 1`] = `
<div
  className="app"
>
  <h1
    className="header"
  >
    Tic Tac Toe
  </h1>
  <Connect(Component) />
  <Connect(Component) />
</div>
`;
